pkg/morph/client/constructor.go

```diff
@@ -54,6 +54,18 @@ type cfg struct {
 
 const (
 	defaultDialTimeout = 5 * time.Second
+
+	// Mostly it's a single event per transaction, with 1s blocks we're
+	// not likely to have more than 5K transactions in a block now.
+	// The default is 512 tx.
+	notifyChanBuf = 5000
```

Comment:
`notifyBufSize` or `notifyChanCap` to not think about "buffer of channels"

Reply:
s/buf/cap/ then